### DISCUSSION TITLE
No history / favorites

### DISCUSSION THREAD

**Discussion by KL7000F** [rustdesk/rustdesk#9122] [General]
### Bug Description

No established connections are saved in the history. I can't save any connections as favorites either.
The peers are available in AppData/Roaming as a *.toml file.

### How to Reproduce

New installation or existing installation.

### Expected Behavior

Established connections are displayed in the history. Favorites can be created.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.1.9

### Screenshots

![Unbenannt](https://github.com/user-attachments/assets/d5684ae5-e417-4c02-8ca3-1da27e076335)

**Comment by cleloo** [comment]
I also encountered it. When I first opened it, there were history and favorites, but after switching twice, they disappeared

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9475
